dependencies.toml

```diff
@@ -981,6 +982,9 @@ javadocs = "https://developers.curioswitch.org/apidocs/java/"
 module = "io.projectreactor:reactor-core"
 version.ref = "reactor"
 javadocs = "https://projectreactor.io/docs/core/release/api/"
+[libraries.context-propagation]
+module = "io.micrometer:context-propagation"
+version.ref = "context-propagation"
```

Comment:
Could be moved up so `[libraries.*]` are sorted alphabetically

Reply:
I see!
I modified it completely.